src/shared/avl/subscribe.ts

```diff
@@ -66,12 +65,12 @@ export const generateSubscriptionRequestXml = (
             "@_xmlns:ns3": "http://www.ifopt.org.uk/ifopt",
             "@_xmlns:ns4": "http://datex2.eu/schema/2_0RC1/2_0",
             SubscriptionRequest: {
-                ...verifiedSubscriptionRequest.SubscriptionRequest,
+                ...subscriptionRequestJson.SubscriptionRequest,
                 VehicleMonitoringSubscriptionRequest: {
-                    ...verifiedSubscriptionRequest.SubscriptionRequest.VehicleMonitoringSubscriptionRequest,
+                    ...subscriptionRequestJson.SubscriptionRequest.VehicleMonitoringSubscriptionRequest,
                     VehicleMonitoringRequest: {
                         "@_version": "2.0",
-                        ...verifiedSubscriptionRequest.SubscriptionRequest.VehicleMonitoringSubscriptionRequest
+                        ...subscriptionRequestJson.SubscriptionRequest.VehicleMonitoringSubscriptionRequest
                             .VehicleMonitoringRequest,
                     },
                 },
```

Comment:
This can be constructed in the same way as the recent change (add the version attribute afterwards)